Invalid format string: array.access.unsafe.high.range

The error message for array.access.unsafe.high.range is an invalid format string. Probably `%d` is not allowed.

Code example:
```java
import org.checkerframework.common.value.qual.IntRange;

public class AccessUnsafeMessage {
	public int m(int[] a, @IntRange(from = 0, to = 12) int i) {
		return a[i];
	}
}
```
Output of the Index Checker:
```
AccessUnsafeMessage.java:5: error: Invalid format string: "[array.access.unsafe.high.range] Potentially unsafe array access: the index could be larger than the array's bound
		return a[i];
		         ^
  index type found: %s
  array type found: %s
  required        : index of type @IndexFor("%s") or @LTLengthOf("%s"), or array of type @MinLen(%d)" args: [@IntRange(from=0, to=12) int, @UnknownVal int @UnknownVal [], a, a, 13]
1 error
```
Expected:
```
AccessUnsafeMessage.java:5: error: [array.access.unsafe.high.range] Potentially unsafe array access: the index could be larger than the array's bound
		return a[i];
		         ^
  index type found: @IntRange(from=0, to=12) int
  array type found: @UnknownVal int @UnknownVal []
  required        : index of type @IndexFor("a") or @LTLengthOf("a"), or array of type @MinLen(13)
1 error
```

--- a/checker/src/main/java/org/checkerframework/checker/index/upperbound/messages.properties
+++ b/checker/src/main/java/org/checkerframework/checker/index/upperbound/messages.properties
@@ -1,5 +1,5 @@
 ### Error messages for the Upper Bound Checker
 array.access.unsafe.high=Potentially unsafe array access: the index could be larger than the array's bound\nfound   : %s\nrequired: @IndexFor("%s") or @LTLengthOf("%s") -- an integer less than %s's length
 array.access.unsafe.high.constant=Potentially unsafe array access: the constant index %s could be larger than the array's bound\nfound   : %s\nrequired: @MinLen("%s") -- an array guaranteed to have at least %s elements
-array.access.unsafe.high.range=Potentially unsafe array access: the index could be larger than the array's bound\nindex type found: %s\narray type found: %s\nrequired        : index of type @IndexFor("%s") or @LTLengthOf("%s"), or array of type @MinLen(%d)
+array.access.unsafe.high.range=Potentially unsafe array access: the index could be larger than the array's bound\nindex type found: %s\narray type found: %s\nrequired        : index of type @IndexFor("%s") or @LTLengthOf("%s"), or array of type @MinLen(%s)
 different.length.sequences.offsets=If offsets are provided, the annotation must contain the same number of sequences and offsets, but this annotation has %s sequence(s) and %s offset(s).

--- a/checker/tests/index/Issue1984.java
+++ b/checker/tests/index/Issue1984.java
@@ -1,0 +1,11 @@
+// Test case for Issue 1984
+// https://github.com/typetools/checker-framework/issues/1984
+
+import org.checkerframework.common.value.qual.IntRange;
+
+public class Issue1984 {
+    public int m(int[] a, @IntRange(from = 0, to = 12) int i) {
+        // :: error: (array.access.unsafe.high.range)
+        return a[i];
+    }
+}

